### ISSUE TITLE
Alert not displaying

### ISSUE THREAD

**Issue by rigaldamez** [ArtSabintsev/Siren#331]
I have just installed `Siren` version 5.4.0 using `pod 'Siren'` in my pod file. 

I have an app version 1.5 which was last published in the appStore approx 2 months ago.

The version details in my xcode project file are:

Version: 0.0.5
Build: 1

(AppStore published version: 1.5)

When testing I change to `Version 0.0.1` then build/run the app and no alert pops up.

I am testing using the provided sample in my `AppDelegate.swift` file

`import Siren`

`var window: UIWindow?`

```
func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {

window?.makeKeyAndVisible()
defaultExampleUsingCompletionHandler()

//...other code eg; Firebase, Facebook initialisation code, present login VC if currentUser == nil

}
```

```
private extension AppDelegate {

    // The simplest implementation of Siren.
    // All default rules are implemented and the
    // results of the completion handler are ignored.
    func defaultExample() {
        Siren.shared.wail()
    }

    // The simplest implementation of Siren.
    // All default rules are implemented and the
    // results of the completion handler are returned or an error is returned.
    func defaultExampleUsingCompletionHandler() {
        
        print("defaultExampleUsingCompletionHandler")
        
        Siren.shared.wail { results in
            switch results {
            case .success(let updateResults):
                print("AlertAction ", updateResults.alertAction)
                print("Localization ", updateResults.localization)
                print("Model ", updateResults.model)
                print("UpdateType ", updateResults.updateType)
            case .failure(let error):
                print(error.localizedDescription)
            }
        }
    }
    
}//end extension

```

**Comment by ArtSabintsev** [comment]
Hi, could you provide me with your bundle identifier so I can test this in the sample app? Also, feel free to edit the example xcode project that's in this repo and see if works for you there as that's what I plan to do with your bundle ID once you provide it.

Thanks!

**Comment by rigaldamez** [comment]
Hi, here is my bundle id: com.rigaldamez.vipeeps

I downloaded the sample project, tried to build and run but got tens of errors :/

**Comment by ArtSabintsev** [comment]
What version of Xcode are you using?

**Comment by rigaldamez** [comment]
I'm using the following;

Xcode: 11.2.1 
ios Deployment Target: 13.0
Swift: 5.0

**Comment by ArtSabintsev** [comment]
Ah, that may be why. Please upgrade to the latest version of Xcode, 11.4.1.

**Comment by rigaldamez** [comment]
I just updated to Xcode 14.4.1 and it's still not displaying the Alert. I see `defaultExampleUsingCompletionHandler` debug line being printed out to the console so the code reached the function but not alert

**Comment by ArtSabintsev** [comment]
ok, per the log:

```
[Siren Error] Error retrieving App Store data as the JSON results were empty. Is your app available in the US? If not, change the `countryCode` variable to fix this error.
```

Is your app available in the US App Store? If not, please do the following:

```
let siren = Siren.shared
siren.apiManager = APIManager(countryCode: "RU")
```

See this example for more information: https://github.com/ArtSabintsev/Siren/blob/ca3559cd93f7c496d10e21ca2b24d1611e7ff75c/Example/Example/AppDelegate.swift#L224-L242

Does this help?

**Comment by rigaldamez** [comment]
Yep! this has worked.

Changing it to the correct `countryCode` worked. thanks for your assistance and for providing such a useful library :)

**Comment by rigaldamez** [comment]
just a quick follow-up, I have changed the version to `1.5` in Xcode , made it `globalRules: .annoying`, should the alert still pop-up even though the version in my project matches the version published in the AppStore?

**Comment by ArtSabintsev** [comment]
hey - happy to help! Glad it worked!

It should not still pop up if the versions are matching exactly; meaning `1.5` is in the app store. Is the version in the app store `1.5` or `1.5.0`?

**Comment by ArtSabintsev** [comment]
I looked at the unit tests, and I test that case. It shouldn't happen. What country code are you using, so I ca replicate this on my end for your app.

**Comment by rigaldamez** [comment]
AppStore is version `1.5 `

Xcode is `Build 1`,  `Version 0.0.5`

Country code: `AU`

**Comment by ArtSabintsev** [comment]
I don't see this issue using the bundle ID that's in the example app (which references the App Store Connect app) and the latest version at this time (1.4.2). 

What's your bundle ID again?

**Comment by rigaldamez** [comment]
hmm bundle is: `com.rigaldamez.vipeeps`

**Comment by ArtSabintsev** [comment]
I did the following just now in Xcode 11.4.1 and Siren v5.4.0 using Swift 5.0 (probably 5.2, since you cant set that directly to my knowledge).

```
bundleID: com.rigaldamez.vipeeps
version: 1.5
build: 1 (this doesn't matter)
countryCode: "AU"
alertType: .annoying
```

With 1.4, I saw the alert. With 1.5 I did not. I did this with many different numbers and was not able to reproduce this issue. 

I can't reproduce this on my end with the sample project. Everything seems to work as expected on my end.
